site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
 - Tests should be written to verify the feature flag works as expected. Note that in cases where there may be a migration or new data, off to on and on to off should both be tested.
 - Log messages by the feature should include the feature flag tag, with the feature flag name as a value, in order to ease debugging.
 
-## Deploying to split.io
+## Split
 
-Deployments to the management system are overseen by the Cloud team. If you have any questions or need help with the process, please ask in the Cloud channel.
+[Split](https://split.io) is the feature flag management service we use for internal testing and our production Cloud service. It enables us to create and control feature flags for multiple servers and environments as well as the ability to run A/B split tests.
 
-1. When ready to deploy the feature, create the feature flag (called a split) in split.io. The name of the feature flag should match the name of the split. Everything else can be left at defaults.
-2. Once created, set the treatment values appropriately. The defaults of "on" an "off" can work for most feature flags.
-3. When ready to deploy to cloud, set the targeting rules appropriately to slowly roll out as required. 
+We've used their SDK to integrate with the Mattermost server. You can find the [Split documentation here](https://help.split.io/hc/en-us).
+
+### Who has access to Split?
+
+We currently have a limited number of seats for our Split account so not everyone gets access. Currently, 2 people per engineering team plus some PM/UX folk will have access. If you need to manage some feature flags, ask your team lead who has access and can help you out. It's up to each team to determine who has access.
+
+### How to log in?
+
+Login is controlled via OneLogin. If you have access you will be able to log in through there.
+### Terminology
+
+In Split, there's some terminology you will need to understand:
+
+ - **Split** - A feature flag
+ - **Treatment** - The values a split (e.g. feature flag) can have
+ - **Environment** - Different environments that splits can target such as test, staging and production
+ - **Server** - Single Mattermost workspace
+ - **Segment** - A list of servers that can be targeted by a split
+
+### Adding a split
+
+After you've added the feature flag in code, you'll need to create a split for it.
+
+ 1. Log in to Split.
+ 2. On left sidebar, click "Splits".
+ 3. At the top click "Create split".
+ 4. Fill in the following:
+   1. Enter the name of your feature flag. It must match the name used in code in `model/feature_flags.go`.
+   2. Select "server" as the traffic type.
+   3. Optionally add tags and a description.
+   4. Click "Create".
+
+The split is now created but not targeting anything. The next section covers adding basic targeting rules.
+
+###  Add targeting rules to a split
```

Comment:
During the 101 training, the CSM mentioned people getting tripped up by the hierarchy of targeting rules. E.g. if a server ID is listed as a targeted server in rule 2, they will still get the first treatment if they satisfy the requirements for rule 1.

Not sure if we want to / need to explicitly mention it here.

Reply:
Hmm I'd prefer to keep our docs simple so that there's less to maintain but what about linking out to the split docs for more info for things like this?

Reply:
I don't feel strongly about it, but it was the one common pitfall the CSM mentioned people experience.